requirements.txt

```diff
@@ -1 +1,3 @@
-adc==1.0.0
+adc-streaming>=1.0.2
```

Comment:
Are we now okay with adc-streaming versions >1.0.0?

Reply:
Yeah, I think so. The strict requirements pinning there was added for some reason, which isn't particularly useful here. I think it was generated from doing a `pip freeze` but the package isn't called `adc`, it's `adc-streaming` so the readthedocs bot crashes here.

Reply:
okay cool